packages/admin/admin-generator/src/commands/generate/generateGrid/generateGqlFieldList.ts

```diff
@@ -20,16 +20,11 @@ const recursiveStringify = (obj: FieldsObjectType): string => {
     return ret;
 };
 
-export function generateGqlFieldList({
-    columns,
-}: {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    columns: Array<GridColumnConfig<any> | ActionsGridColumnConfig | VirtualGridColumnConfig<any>>;
-}) {
+export function generateGqlFieldList<T extends { __typename: string }>({ columns }: { columns: Array<GridConfigGridColumnDef<T>> }) {
```

Comment:
Have you considers using `GridConfig["columns"]` instead of creating and exporting an additional type?

Reply:
Did you change this? The commit fb2b1b0cbe50e353fc90be1d6de70fc78079294b does something else.

Reply:
its using `GridConfig<T>["columns"]` now